src/shared/components/common/loading-skeleton.tsx

```diff
@@ -130,60 +107,46 @@ class TrendingCommunitiesLoadingSkeletonItem extends Component<
         <div className="row">
           <div className="col flex-grow-0" style={{ "padding-right": "0px" }}>
             <div
-              className="trending-community-icon-skeleton d-flex skeleton-pulse"
-              data-bs-theme={this.props.theme}
-            ></div>
+              className="d-flex placeholder-glow"
+              style={{ width: "1.5rem", height: "1.5rem" }}
```

Comment:
Same, use bootstrap classes. Also on line 108.

Go through every `style=` in this file, there's a high chance all of them are covered by bootstrap classes.

Reply:
I couldn't find a css class with only "width: 1.5rem" or "width: 1.5em" when searching through the bootstrap  files. I was able to fix the ones with "width: 100%", "height: 100%", and "padding-end: 1.5rem"  though.

Reply:
You could try `w-auto`, and we also already have an `.icon` class that could also work. 

If you absolutely need to add another class just for this, add it to `src/assets/css/main.css` and don't inline this style.